config/settings.yml

```diff
@@ -1372,6 +1372,9 @@ vanotify:
         form_10_7959c_failure_email: form_10_7959c_failure_email
         form_10_7959a_email: form_10_7959a_email
         form_10_7959a_failure_email: form_10_7959a_failure_email
+        pega_team_missing_status_email: pega_team_missing_status_email
+        pega_team_zsf_email: pega_team_zsf_email
+
```

Comment:
These look like the only changes to this file that we need - would it be possible to undo all the other changes? Unless they're necessary